operations/node_management/Swap_a_Compute_Blade_with_a_Different_System_Using_SAT.md

```diff
@@ -0,0 +1,358 @@
+   [Tue Jul 21 13:09:54 2020] DVS: merge_one#351: New node map entry does not match the existing entry
+   [Tue Jul 21 13:09:54 2020] DVS: merge_one#353:   nid: 8 -> 8
+   [Tue Jul 21 13:09:54 2020] DVS: merge_one#355:   name: 'x3000c0s19b1n0' -> 'x3000c0s19b1n0'
+   [Tue Jul 21 13:09:54 2020] DVS: merge_one#357:   address: '10.252.0.26@tcp99' -> '10.252.0.33@tcp99'
+   [Tue Jul 21 13:09:54 2020] DVS: merge_one#358:   Ignoring.
+   ```
+
+1. Make sure the Configuration Framework Service (CFS) finished successfully. Review *HPE Cray EX DVS Administration Guide 1.4.1 S-8004*.
+
+1. SSH to the node and check each DVS mount.
+
+   ```bash
+   nid001133:~ # mount | grep dvs | head -1
+   ```
+
+   Example output:
+
+   ```text
+   /var/lib/cps-local/0dbb42538e05485de6f433a28c19e200 on /var/opt/cray/gpu/nvidia-squashfs-21.3 type dvs (ro,relatime,blksize=524288,statsfile=/sys/kernel/debug/dvs/mounts/1/stats,attrcache_timeout=14400,cache,nodatasync,noclosesync,retry,failover,userenv,noclusterfs,killprocess,noatomic,nodeferopens,no_distribute_create_ops,no_ro_cache,loadbalance,maxnodes=1,nnodes=6,nomagic,hash_on_nid,hash=modulo,nodefile=/sys/kernel/debug/dvs/mounts/1/nodenames,nodename=x3000c0s6b0n0:x3000c0s5b0n0:x3000c0s4b0n0:x3000c0s9b0n0:x3000c0s8b0n0:x3000c0s7b0n0)
+
+   nid001133:~ # ls /var/opt/cray/gpu/nvidia-squashfs-21.3
+   rootfs
+   ```
+
+### Check the HSN for the affected nodes
+
+1. Determine the pod name for the Slingshot fabric manager pod and check the status of the fabric.
+
+   ```bash
+   ncn# kubectl exec -it -n services \
+           $(kubectl get pods --all-namespaces |grep slingshot | awk '{print $2}') \
+           -- fmn_status
+   ```
+
+### <a name="check-dns"></a>Check DNS
+
+1. Check for duplicate IP address entries in the State Management Database (SMD). Duplicate entries will cause DNS operations to fail.
+
+   ```bash
+   ncn# ssh uan01
+   ssh: Could not resolve hostname uan01: Temporary failure in name resolution
+   ncn# ssh x3000c0s14b0n0
+   ssh: Could not resolve hostname x3000c0s14b0n0: Temporary failure in name resolution
+   ncn# ssh x1000c1s1b0n1
+   ssh: Could not resolve hostname x1000c1s1b0n1: Temporary failure in name resolution
+   ```
+
+   The Kea configuration error will display a message similar to the message below. This message indicates a duplicate IP address (`10.100.0.105`) in the SMD:
+
+   ```text
+   Config reload failed
+   [{'result': 1, 'text': "Config reload failed: configuration error using file '/usr/local/kea/cray-dhcp-kea-dhcp4.conf':
+   failed to add new host using the HW address '00:40:a6:83:50:a4 and DUID '(null)' to the IPv4 subnet id '0' for the address 10.100.0.105: There's already a reservation for this address"}]
+   ```
+
+1. Use the following example `curl` command to check for active DHCP leases. If there are 0 DHCP leases, there is a configuration error.
+
+   ```bash
+   ncn# curl -H "Authorization: Bearer ${MY_TOKEN}" -X POST -H "Content-Type: application/json" \
```

Comment:
Need to provide instructions (or link to them) for setting the $TOKEN variable. Possibly add this to the prerequisites list earlier (API token must be stored in $TOKEN variable), with a link to documentation on how to do that. Or just add a step to this document to set it.